### PR TITLE
Blog Page

### PR DESCRIPTION
**Pull request by evanessa234**
medium blog embeded, more styling will be required later issue #9

### CODE REVIEW

File: pages/blog.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import Head from 'next/head';
+import MediumBlogContainer from 'src/components/mediumBlogContainer';
+
+const Blog = () => (
+    <>
+        <Head>
+            <title>DSC-RAIT Medium Blogs</title>
+        </Head>
+        <MediumBlogContainer>
+        <blockquote class="embedly-card" data-card-controls="0"><h4><a href="https://medium.com/developer-student-club-rait">Developer Student Club, RAIT - Medium</a></h4><p>A community of budding developers, entrepreneurs and innovators, getting ready to take on real-world challenges by harnessing the power of technlogy.</p></blockquote>
```
Comment (dwvicy) [inline pages/blog.tsx:10]: Have you installed all the dependencies?

**Review by dwvicy — CHANGES_REQUESTED** [review]
Your commit name should in the format feat: add the blog page

**Comment by dwvicy** [comment]
Also, update the code too with the stuff I just pushed

File: pages/blog.tsx (hunk 1 above)
Comment (evanessa234) [reply pages/blog.tsx:10]: no extra dependency is required for my code

**Pull request closed without merging**